src/client/components/glossary/Glossary.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+
+import Definition from '../definition';
+import definitionData from '../definition/definitionData';
+import {
+  DEFINITION_OPEN,
+  DEFINITION_HIGH,
+  DEFINITION_LOW,
+  DEFINITION_MARKET_CAP,
+  DEFINITION_VOLUME,
+  DEFINITION_DIVIDEND,
+  DEFINITION_PE_RATIO,
+  DEFINITION_ROE,
+  DEFINITION_FCFY
+} from '../../../constants';
+
+const Glossary = () => {
+  const definitions = [
```

Comment:
glossaries are alphabetized

Reply:
this can be achieved using an array approach, as described in a previous comment.